### PR TITLE
Added FAQs related to image migration tool

### PR DESCRIPTION
**Pull request by abhi-harness-cxe**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 1]
```diff
@@ -2552,3 +2552,23 @@ If it is operating as expected, the Kaniko CLI will show the following in the CL
...
+
+Yes, if the source registry is public or does not require authentication, you can omit the username and password for the source.
+
+#### What is the purpose of the insecure option in the plugin settings?
+
+The insecure option, when set to true, disables TLS, which is normally enabled by default to secure the connection.
+
+#### Where can I find more information about using the Image Migration Tool?
+
+Additional details, including configuration examples and usage guidelines, can be found in the plugin’s [README documentation](https://github.com/harness-community/drone-docker-image-migration?tab=readme-ov-file#plugin-image).
```
Comment (dewan-ahmed) [inline kb/continuous-integration/continuous-integration-faqs.md:2574]:
```suggestion
Additional details, including configuration examples and usage guidelines, can be found in the plugin’s [README documentation](https://github.com/harness-community/drone-docker-image-migration?tab=readme-ov-file).
```

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 2]
```diff
@@ -2552,3 +2552,23 @@ If it is operating as expected, the Kaniko CLI will show the following in the CL
...
+
+No, it can only push to registries that support basic authentication, AWS ECR registries, and GAR registries with access token authentication.
+
+#### Is it possible to use the plugin without providing a username and password?
+
+Yes, if the source registry is public or does not require authentication, you can omit the username and password for the source.
+
+#### What is the purpose of the insecure option in the plugin settings?
+
+The insecure option, when set to true, disables TLS, which is normally enabled by default to secure the connection.
```
Comment (dewan-ahmed) [inline kb/continuous-integration/continuous-integration-faqs.md:2570]:
```suggestion
The insecure option, when set to `true`, disables TLS, which is normally enabled by default to secure the connection.
```

**Review by dewan-ahmed — COMMENTED** [review]
@abhi-harness-cxe pls check the review comments.

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**